src/foundry/foundry.js-lenient/globalVariables.d.ts

```diff
@@ -0,0 +1,104 @@
+declare global {
+  /**
+   * The lenient variety of global variables defining everything up front.
+   *
+   * TODO: Add more documentation here to describe as necessary.
+   */
+  let vtt: 'Foundry VTT';
```

Comment:
This doc comment should be either removed or made sure it is not attached to the `vtt` variable.

Reply:
Hi @kmoschcau. Indeed in `src/foundry/index.d.ts` the import of globalVariables.d.ts was not intended and removed. The comment is moved to the top of the lenient globalVaraiables version. 

Regarding where to put the two different versions that is solely up to the maintainers to agree upon the method used. The current method is the "least worst" kind of solution at the time I discussed this addition with @ghost91- from what is otherwise an opinionated decision. At this point it seems reasonable to accept the PR and then make a follow up change for whatever is collectively agreed upon as I have no horse in that race so to speak. As the important part IMHO is what is now possible in `package.json` to easily support two versions of the types.

I am however interested in using the lenient typings in a large library module / NPM package for developers in the coming months, so definitely appreciate if the types expand to supporting a lenient mode.